modules/core/src/main/java/org/apache/synapse/config/xml/MessageStoreMediatorFactory.java

```diff
@@ -70,4 +85,16 @@ protected Mediator createSpecificMediator(OMElement elem, Properties properties)
     public QName getTagQName() {
        return STORE_Q;
     }
+
+    public boolean checkForBraces(OMAttribute atr){
+        if((atr.getAttributeValue().startsWith("{")) && (atr.getAttributeValue().endsWith("}")))
```

Comment:
Missing opening/closing brackets of if-else block.

Reply:
If-condition simplified.